app/src/Commands/Instances/RestartCommand.php

```diff
@@ -0,0 +1,125 @@
+<?php declare(strict_types=1);
+
+/* Copyright (c) 2022 - Daniel Weise <daniel.weise@concepts-and-training.de> - Extended GPL, see LICENSE */
+
+namespace CaT\Doil\Commands\Instances;
+
+use CaT\Doil\Lib\Posix\Posix;
+use CaT\Doil\Lib\Docker\Docker;
+use CaT\Doil\Lib\ConsoleOutput\Writer;
+use CaT\Doil\Lib\FileSystem\Filesystem;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class RestartCommand extends Command
+{
+    protected static $defaultName = "instances:up";
```

Comment:
This should be 'instances:restart' afaik

Reply:
done!